lib/spack/spack/cmd/style.py

```diff
@@ -242,12 +252,9 @@ def run_flake8(flake8_cmd, file_list, args):
     # run in chunks of 100 at a time to avoid line length limit
     # filename parameter in config *does not work* for this reliably
     for chunk in grouper(file_list, 100):
-
         output = flake8_cmd(
-            # use .flake8 implicitly to work around bug in flake8 upstream
-            # append-config is ignored if `--config` is explicitly listed
-            # see: https://gitlab.com/pycqa/flake8/-/issues/455
-            # "--config=.flake8",
+            # always run with config from running spack prefix
+            "--config=%s" % os.path.join(spack.paths.prefix, ".flake8"),
```

Comment:
Should these configs check for existence before running the commands (and pyproject.toml too?)

Reply:
good point -- added